velox/exec/tests/SpillerTest.cpp

```diff
@@ -1040,6 +1045,51 @@ class AllTypes : public SpillerTest,
   }
 };
 
+TEST_P(AllTypes, nonSortedSpillFunctions) {
+  struct {
+    uint64_t maxSpillRunRows;
+
+    std::string debugString() const {
+      return fmt::format("maxSpillRunRows {}", maxSpillRunRows);
+    }
+  } testSettings[] = {{101}, {0}};
+
+  for (const auto& testData : testSettings) {
```

Comment:
Do we need this test? I assume non-sorted spiller will ignore the max spill run parameter except hash join spiller type? Can we just verify that the number of files spilled by hash join spiller is controlled by this parameter? Like targetFileSize is extreme large?

Reply:
Can you add to test a large value per our discussion? Thanks!